acceptance/api/v1/helpers_test.go

```diff
@@ -126,7 +126,7 @@ func waitForStaging(stageID string) {
 	Eventually(func() string {
 		out, err := proc.Kubectl("get", "job",
 			"--namespace", helmchart.StagingNamespace,
-			stageID,
+			"-l", fmt.Sprintf("epinio.suse.org/stage-id=%s", stageID),
```

Comment:
note; I see this wasn't a constant before. Would be nice to have labels somewhere, so we know which labels we use and only need to make changes in one place. However we don't want to introduce to many package imports..

Reply:
I switched this back to a direct name again, constructed in the caller.
Making consts from the labels I see as separate cleanup issue.